### PR TITLE
feat(*):Migrating apps to android 9

### PR DESCRIPTION
**Pull request by snooky23**
## Description
Update Android plugins to android 9 and latest Applicaster changes.

## Checklist

* [X ] PR is scoped to one task
* [ ] Tests are included
* [ ] Documentation is included

* [ ] This PR is not making any UI change
* [ ] This PR is making a UI change
  * [ ] Screenshot(s) included

## QA :

* required test cases:
* specific apps / plugins to test :

### CODE REVIEW

File: Advertisement/Android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#Fri Jun 01 16:48:12 EDT 2018
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```
Comment (giltab) [inline Advertisement/Android/gradle/wrapper/gradle-wrapper.properties]: we use 4.5.1

File: FullScreen/android/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Fri Mar 23 09:24:10 CET 2018
+#Fri Jun 01 16:48:12 EDT 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```
Comment (giltab) [inline FullScreen/android/gradle/wrapper/gradle-wrapper.properties]: 4.5.1

File: VideoPlayer/Android/app/build.gradle [hunk 3]
```diff
@@ -1,17 +1,21 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 27
+    compileSdkVersion 28
     defaultConfig {
         applicationId "com.applicaster.player.plugins.zapppluginplayerexampleandroid"
-        minSdkVersion 19
-        targetSdkVersion 27
+        minSdkVersion 26
```
Comment (giltab) [inline VideoPlayer/Android/app/build.gradle]: 19

File: Advertisement/Android/gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (snooky23) [reply Advertisement/Android/gradle/wrapper/gradle-wrapper.properties]: Done

File: FullScreen/android/gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (snooky23) [reply FullScreen/android/gradle/wrapper/gradle-wrapper.properties]: done

File: VideoPlayer/Android/app/build.gradle (hunk 3 above)
Comment (snooky23) [reply VideoPlayer/Android/app/build.gradle]: done

**Review by giltab — APPROVED** [review]

**Pull request merged**